solr/core/src/test/org/apache/solr/search/TestExtendedDismaxParser.java

```diff
@@ -1771,6 +1775,35 @@ public void testSplitOnWhitespace_Basic() throws Exception {
     assertThat(parsedquery, anyOf(containsString("((name:stigma | title:stigma))"), containsString("((title:stigma | name:stigma))")));
   }
 
+    @Test 
+    public void testQf_stringFieldWithSowFalse_shouldBuildSingleQueryClause() throws Exception
+    {
+        assertJQ(req("qf", "trait_ss", "defType", "edismax", "q", "multi term", "sow", "false"),
+            "/response/numFound==1", "/response/docs/[0]/id=='75'");
+
+        String parsedquery = getParsedQuery(
+            req("qf", "trait_ss", "q", "multi term", "defType", "edismax", "sow", "false", "debugQuery", "true"));
+        assertThat(parsedquery, anyOf(containsString("((trait_ss:multi term))")));
```

Comment:
disjunction `anyOf(...)` shouldn't be necessary to wrap a single clause -- can just use `containsString(...)` directly at the top level.

Reply:
Done!